github/Repository.py

```diff
@@ -1867,6 +1867,22 @@ def get_forks(self):
             None
         )
 
+    def create_fork(self, organization=None):
+        """
+        :calls: `POST /repos/:owner/:repo/forks <http://developer.github.com/v3/repos/forks>`_
```

Comment:
Please document organiztion in the docstring.

Unset parameters must default to NotSet (see elsewhere in that file)

Reply:
thanks will do

Reply:
Just thinking about it more -- is it really organization, or is it destination of the fork?

Reply:
I just got the name from the GitHub API docs
![image](https://user-images.githubusercontent.com/47873678/69796158-a1715000-1221-11ea-80c6-9c097f0bb92d.png)

I think GitHub often refers to a collection of repositories as an organization even though sometimes this is a single user.

Reply:
I guess if not specified then it is your user account otherwise you are forking into an organization - as presumably you can't fork into someone else user account.